src/Interfacer.jl

```diff
@@ -0,0 +1,75 @@
+"""
+    Interfacer
+
+...
+"""
```

Comment:
There are some missing docstring warnings in the docs
![image](https://github.com/CliMA/ClimaCoupler.jl/assets/39932030/bcc609a6-fdaf-4f02-8d2d-fc172cc9d380)